numpy/_core/numeric.pyi

```diff
@@ -109,7 +109,7 @@ from numpy._typing import (
     _ArrayLikeComplex_co,
     _ArrayLikeTD64_co,
     _ArrayLikeObject_co,
-    _ArrayLikeUnknown,
+    _NestedSequence,
```

Comment:
```suggestion
    _ArrayLikeUnknown,
    _NestedSequence,
```

Reply:
it's because https://github.com/numpy/numpy/pull/28733 hasn't been backported (which is a good thing)